common/rabbitmq/values.yaml

```diff
@@ -122,3 +122,19 @@ customConfig:
   # to set a custom limit please use the following format: 50MB or 1GB
   # if not set default value of 50MB will be used
   # disk_free_limit.absolute: 500MB
+
+prometheus-monitors:
+  serviceMonitor:
+    enabled: true
+  prometheus: "openstack"
+  matchLabels:
+    app: rabbitmq
```

Comment:
I think because we are giving the selector in value it is not a good idea to use sub-chart 
beacuse this selector needs to be dynamic value
`app: {{ template "fullname" . }}`
and because you can not template values in Helm so i think we need to avoind using the prometheus-monitors subchart and create `ServiceMonirot/PodMonitor ` resource in rabbitmq helmchart